src/IECoreImage/ClientDisplayDriver.cpp

```diff
@@ -73,7 +80,11 @@ class ClientDisplayDriver::PrivateData : public RefCounted
 			m_socket.close();
 		}
 
+#if BOOST_VERSION >= 106600
+		boost::asio::io_context m_service;
+#else
 		boost::asio::io_service m_service;
+#endif
```

Comment:
I think this can be simpler. `io_context` is available in 1.80 (the version we're building with), and `io_service` is just a typedef referencing it. So I think we can just change to `io_context` unconditionally, and have one version of the code that works for both boost versions.